packages/fast-animation/CHANGELOG.md

```diff
@@ -1,43 +1,44 @@
-# Change Log
+# Change log
```

Comment:
I believe this file is auto-generated on publish oof new package versions.  Not sure if there is a way to format based on our standards rather than conventional commits.

Reply:
+1 - we can look into configuring conventional-commits but I would't  recommend changing changelogs directly.

Reply:
Just had a chat with Nick. The issue was the change logs were inconsistent switching heading size, linking and even some bugs in sub-headings. This resulted in the pages looking broken because of a number of the changes or not following consistent patterns in heading case. Given the inconsistencies from one part of the change log to the other, it appears Lerna a) is appending new updates and b) has had some kind of change to its template somewhere around version 3. Based on this, I propose committing the changes and creating another task to look at updating the Lerna templates.